envoy/adapter/adapter.go

```diff
@@ -257,44 +279,6 @@ func connectionManagerForService(svc *service.Service, envoyServiceName string)
 				Cluster: envoyServiceName,
 			},
 		}
-	case "ws":
```

Comment:
You removed websocket support it looks like?

Reply:
Nope, IIRC I just deduplicated the code. See [here](https://github.com/NinesStack/sidecar/blob/6a309151b1de50830b0a985380ad77fc1ba70c76/envoy/adapter/adapter.go#L229-L230) and [here](https://github.com/NinesStack/sidecar/blob/6a309151b1de50830b0a985380ad77fc1ba70c76/envoy/adapter/adapter.go#L266-L272).